<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>CamGui::Cutting</class>
 <widget class="QDialog" name="CamGui::Cutting" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>240</y>
     <width>69</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Master-Radius</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>270</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Slave-Radius</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>320</y>
     <width>155</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Correction Factor for Springback</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>350</y>
     <width>155</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Limit Angle</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayout_3" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>10</y>
     <width>211</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="CalculcateZLevel" >
      <property name="text" >
       <string>Calculate Z-Level Toolpath</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3" >
      <property name="text" >
       <string>Calculate Feature Based Toolpath</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton" >
      <property name="text" >
       <string>Make a Best-Fit</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4" >
      <property name="text" >
       <string>Correct Springback</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5" >
      <property name="text" >
       <string>Approximate NURB Surface</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayout" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>131</width>
     <height>140</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="toolpath_calculation_highest_level_button" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>Highest Level</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="toolpath_calculation_middle_level_button" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>Middle Level</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="toolpath_calculation_lowest_level_button" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>Lowest Level</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="toolpath_calculation_go_button" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>GO</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayout_4" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>50</y>
     <width>131</width>
     <height>140</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="best_fit_cad_button" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>CAD-Part</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="best_fit_fe_button" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>FE-Result</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="best_fit_mesh_button" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>Scanned Mesh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="best_fit_go_button" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>GO</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>20</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-style:italic;">Best-Fit&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-size:12pt; font-style:italic;">Tool-Path-Calculation&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>300</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cutting-Distance</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayout_2" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>210</y>
     <width>53</width>
     <height>184</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox" />
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2" />
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_3" />
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_4" />
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_5" />
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>390</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton" >
      <property name="text" >
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
